webapp-django/crashstats/settings/base.py

```diff
@@ -768,9 +767,30 @@ def JINJA_CONFIG():
         #     'ELASTICSEARCH_DOCTYPE',
         #     'crash_reports'
         # )
-    }
+    },
+    'database_password': config(
```

Comment:
Why isn't this all in a ``postgres`` or ``database`` namespace, like ``elasticsearch`` stuff is?